scanner.py

```diff
@@ -329,6 +329,13 @@ def __init__(self, config: ConfigWrapper):
             desc=self.cmd_Z_OFFSET_APPLY_PROBE_help,
         )
 
+    def is_model(self, gcmd: GCodeCommand):
+        model = self.model
+        if model is None:
+            raise gcmd.error(
+                "No model currently selected - make sure you have run CARTOGRAPHER_CALIBRATE first"
```

Comment:
Should this link to the docs like we do for the validate?

Reply:
Done